test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -177,17 +222,14 @@ public void testInitOfUrlMetadataListerDocIdIsUrlTrue() throws Exception {
 
   private Config createStandardConfig(Map<String, String> moreEntries) {
     Map<String, String> configEntries = new HashMap<String, String>();
-    // driverClass must be specified, but (other than it pointing at a valid
-    // class), the value does not matther.
-    configEntries.put("db.driverClass",
-        "com.google.enterprise.adaptor.database.DatabaseAdaptor");
-    configEntries.put("db.url", "must be set");
-    configEntries.put("db.user", "must be set");
-    configEntries.put("db.password", "must be set");
+    configEntries.put("db.driverClass", JdbcFixture.DRIVER_CLASS);
+    configEntries.put("db.url", JdbcFixture.URL);
+    configEntries.put("db.user", JdbcFixture.USER);
+    configEntries.put("db.password", JdbcFixture.PASSWORD);
     configEntries.put("db.uniqueKey", "must be set:string");
-    configEntries.put("db.everyDocIdSql", "must be set");
+    configEntries.put("db.everyDocIdSql", "select 42 as \"must be set\"");
```

Comment:
If we exclude the connection properties, there are only 4 required properties (uniqueKey, everyDocIdSql, singleDocContentSql, and modeOfOperation). I don't think it's unreasonable for each test to set those. I am interested in getting all of the tests to the point where we don't have to specify the default values, but I don't know if I want to go past that. I do, as I mention in the commit message, want to revisit the suppressions once we get there. I started fixing them so the tests pass with them, but it was a complicated can of worms, and I wanted to rethink it once each test was only specifying the necessary properties.